### ISSUE TITLE
x/mobile/cmd/gomobile: download `adb` for install

### ISSUE THREAD

**Issue by techtonik** [golang/go#17481]
### What did you expect to see?
Expected `gomobile init` to download `adb` for `gomobile install`.

### What did you see instead?
Need to download/unpack and add `adb` to path manually.

**Comment by crawshaw** [comment]
I did want to do this, but there's no easy reliable source of adb to download. Ideally the protocol would be simple enough that we could implement it, but it's not.

As I don't know how to do, I'm closing this.

**Comment by techtonik** [comment]
Why there is no reliable source? `platform-tools_r24.0.3-windows.zip` from any update mirror, like http://mirrors.neusoft.edu.cn/android/repository/

**Comment by crawshaw** [comment]
When I looked into it there were no obvious rules to follow for versioning. I decided it was easier to say "install android studio".

**Comment by techtonik** [comment]
Do you mean that Eclipse update site is hard to parse to find archives like `platform-tools_*.zip`? Or did I miss something else?

**Comment by crawshaw** [comment]
I mean, in ```platform-tools_*.zip```, what goes in the *? There are lots of decisions we could make (a fixed vetted value, the latest, etc), but no obvious one. I would much rather say "use Android Studio".

**Comment by techtonik** [comment]
And what is the `*` in the NDK? I guess it is just current version, no?
